subprojects/model-core/src/test/groovy/org/gradle/api/internal/provider/WithSideEffectProviderTest.groovy

```diff
@@ -86,87 +99,7 @@ class WithSideEffectProviderTest extends Specification {
         "getOrElse" | { it.getOrElse(88) }
     }
 
-    def "does not run side effect when calling '#method' on missing value provider"() {
```

Comment:
These removed tests are covered with identical generalized tests in the `ProviderSpec`